vignettes/dm-introduction-relational-data-models.Rmd

```diff
@@ -132,7 +130,8 @@ Which columns of the `airlines` table can serve as a primary key?
 
 ```{r}
 dm %>% 
-  cdm_enum_pk_candidates(airports)
+  cdm_enum_pk_candidates(airports) %>% 
+  arrange(desc(candidate))
```

Comment:
I'm confused -- why do we arrange here but not below?

Reply:
That's because `cdm_enum_fk_candidates()` arranges the tibble and `cdm_enum_pk_candidates()` doesn't. But I agree, we should have consistent behaviour.

I will change this for `cdm_enum_pk_candidates()` in another PR, if that's ok.

Reply:
Let's do it in this PR.

Reply:
`cdm_enum_pk_candidates()` arranges the output in an equivalent way to `cdm_enum_fk_candidates()` now